hs-ucan/library/Web/DID/Types.hs

```diff
@@ -157,10 +146,16 @@ parseText txt =
                 CryptoFailed cryptoError -> fail $ "Unable to parse Ed25519 key: " <> show cryptoError
                 CryptoPassed edPK -> return $ Ed25519PublicKey edPK
 
+        (0x85 : 0x24 : rsaASNPublicKeyW8s) ->
+          case Public.decodeASN1DERRSAPublicKey $ BS.pack rsaASNPublicKeyW8s of
+            Right pk -> pure $ RSAPublicKey pk
+            Left err -> fail err
+
+        -- Backwards compatibility
         (0x00 : 0xF5 : 0x02 : rsaKeyW8s) ->
           RSAPublicKey <$> parseKeyW8s (BS64.encode $ BS.pack rsaKeyW8s)
```

Comment:
This is what I'm talking about: We just add the ability to interpret another multiformat prefix.